core/src/main/java/org/jdbi/v3/core/statement/PreparedBatch.java

```diff
@@ -22,6 +22,7 @@
 import java.util.Map;
 import java.util.NoSuchElementException;
 
+import edu.umd.cs.findbugs.annotations.SuppressWarnings;
```

Comment:
This import needs to be removed, it changes the SuppressWarnings down below from the java lang one to the findbugs one.

Reply:
Got it. Weird, dunno how it got there in the first place.

Reply:
probably IDE auto-import